internal/cluster/cluster_test.go

```diff
@@ -26,7 +26,7 @@ import (
 	"github.com/submariner-io/submariner-operator/internal/cluster"
 )
 
-var _ = Describe("IsValidID", func() {
+var _ = Describe("TestClusterIDs", func() {
```

Comment:
I think we should keep this as "IsValidID" and add a separate `Describe` for "SanitizeD".